src/index.html

```diff
@@ -5,9 +5,300 @@
+              Pointillism, which originated in the music of the early twentieth century, microforms, found a distant historical.
+            </p>
+          </div>
+
+          <form
+            class="consultation__form form"
+            action="#"
+            method="post"
+          >
+            <input
+              class="form__item"
+              type="text"
+              name="name"
+              placeholder="Name"
+              required
+            >
+            <input
+              class="form__item"
+              type="tel"
+              pattern="^((8|\+3)[\- ]?)?(\(?\d{3}\)?[\- ]?)?[\d\- ]{7,10}$"
+              placeholder="Phone"
+              name="phone"
+              required
+            >
+            <input
+              class="form__item"
+              type="email"
+              placeholder="Email"
+              name="email"
+              required
+            >
+            <button type="submit" class="form__btn button">Consultation</button>
+          </form>
+        </div>
+      </section>
+      <section class="testimonials" id="testimonials">
+        <h1 class="heading">Testimonials</h1>
+
+        <div class="testimonials__persons ">
+          <div class="person">
+            <img
+              class="person__photo"
+              src="./images/people/1.jpg"
+              alt="tim-cook"
+            >
+            <div class="person__name"> Tim Cook</div>
+            <div class="person__status">CEO</div>
+            <div class="person__about">Nice work, Certificates National Association of Realtors (USA)</div>
+          </div>
+          <div class="person">
+            <img
+              class="person__photo"
+              src="./images/people/2.jpg"
+              alt="Katherine-Adams"
+            >
+            <div class="person__name">Katherine Adams</div>
+            <div class="person__status">Senior Vice President</div>
+            <div class="person__about">My soul is illuminated by an unearthly joy, as these beautiful spring morning, which I enjoy with all my heart</div>
+          </div>
+          <div class="person">
+            <img
+              class="person__photo"
+              src="./images/people/3.jpg"
+              alt="Jonathan-Ive"
+            >
+            <div class="person__name">Jonathan Ive</div>
+            <div class="person__status">Chief Design Officer</div>
+            <div class="person__about">When my lovely valley of steam rises and half-day sun is above an impermeable</div>
+          </div>
+        </div>
+      </section>
+      <section class="contact" id="contacts">
+        <h1 class="heading">Contact us</h1>
+        <div class="contact__wrapper">
+          <div class="contact__info">
+            <div class="contact__connecting-info">
+              <p class="contact__title">Phone</p>
+
+              <p class="contact__item">
+                <a class="contact__link" href="tel:+1 234 555-55-55">+1 234 555-55-55</a>
```

Comment:
```suggestion
                <a class="contact__link" href="tel:+1 234 555-55-55">
                  +1 234 555-55-55
                </a>
```

Якщо можна зробити ще трішечки читабельніше, краще завжди не відмовлятися від такої можливості